native/dll/Win32DLL/SageTVWin32DLL.cpp

```diff
@@ -979,7 +979,11 @@ void loadAWTLib()
 		strcat_s(includedJRE, sizeof(includedJRE), "jre\\bin\\jawt.dll");
 		sageLoadedAwtLib = LoadLibrary(includedJRE);
 		if (sageLoadedAwtLib)
+		{
+			slog(("INFO:jawt.dll path = %s\r\n", includedJRE));;

```

Comment:
Can you remove the extra parenthesis and semicolon on this one and the other slog line you added below? Or just remove those log lines...either way. Then this one will be done. :)

Reply:
Done semicolon.
Without the extra parenthesis it complains "cannot determine which instance of overloaded function 'sysOutPrint' is intended"
Does that mean I'm missing something else somewhere?

Reply:
If it's saying that then you can leave them, it's because of the macro...I forgot that's how we did it.

Reply:
Done